### DISCUSSION TITLE
Personalized Report Card

### DISCUSSION THREAD

**Discussion by Priyanka-droid** [scaleracademy/open-source-projects#80] [Ideas]
**Overview**
Most of the candidates are confused about when are they really prepared to give interviews. This assessment-based report card may help them solve this problem and prepare them at a higher pace by telling them their personalized report and weak points.

**Idea**
Creating a system where candidates would be given a few tests with different difficulties and they will be scored based on those tests and all their weak points will be mentioned in their report cards and resources as well as tests will be provided along with that to prepare those topics. It will be a personalized report card depending on the score of the person on each topic. If a candidate has to work more on a certain algorithm, he/she will be provided with more quality problems of that certain topic. 

- Other than this assessment there should be an assessment of communication and posture as well. The system will be rating the person based on his/her communication skill. For example, a candidate can be given a real-time problem and will be asked to explain that within a given time. And the system will be checking their posture, facial expressions, and communication throughout the assessment and the candidate will be scored based on that. This way they can know their weak points and prepare according to them. 

- And it would also be good if they could know how much they are prepared for a certain position based on previous records of the questions asked in that company.

**Comment by championswimmer** [comment]
Interesting idea, at Scaler in fact we just started something similar here 
https://www.scaler.com/event/know-where-you-stand-free-self-assessment-test
